rules/apm/apm_403_response_to_a_post.toml

```diff
@@ -1,6 +1,7 @@
 [metadata]
 creation_date = "2020/02/18"
 maturity = "production"
+min_stack_version = "7.13.0"
```

Comment:
You okay backing these changes out so this PR is code only?
I'm not sure if we want these set to 7.13 or not, we can automatically anchor them if they are unset later.

I would rather leave it empty if we're unsure

Reply:
5c53fc95354e7dcb68c9d395a26ea7c2af62b9f4